src/index.html

```diff
@@ -11,5 +11,67 @@
     <header class="app-header">
       <h1 class="app-header__heading">Quiz App</h1>
     </header>
+    <main class="app-main">
+      <section class="quiz-card">
+        <h2>Waht is the question?</h2>
```

Comment:
Just a small typo :)
```suggestion
        <h2>What is the question?</h2>
```